modules/heat_conduction/include/materials/GapConductance.h

```diff
@@ -88,6 +88,7 @@ class GapConductance : public Material
   const VariableValue & _gap_temp_value;
   MaterialProperty<Real> & _gap_conductance;
   MaterialProperty<Real> & _gap_conductance_dT;
+  MaterialProperty<Real> & _gap_conductivity_property;
```

Comment:
I would use _gap_conductivity as the member variable name but this name is already used in this class. I could use _gap_thermal_conductivity as the name instead.